client/blocks/app-banner/style.scss

```diff
@@ -3,7 +3,7 @@
 	bottom: 0;
 	width: 100%;
 	background-color: var( --color-primary-light );
-	color: var( --color-primary-dark );
+	color: $white;
```

Comment:
Given that the `background-color` is using a CSS variable, wouldn't we want to use a CSS variable here as well?

Reply:
I think there's still a few open questions on how we're going to approach text color thinking of Laser Black which is why ideally I'd like to defer that question for now and do those in a separate step as a whole. Would that work?

Thinking of naming what would be an appropriate name for such a variable - maybe `--color-text-light`?

Reply:
I added --color-white in a separate PR that has now landed. We could use that for these cases for now... I agree that we'll have to revisit this later when we go after the dark theme. --color-text-light sounds like a good idea.